docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,493 @@
+    alias: <optional alias>
+configuration:
+  allow_experimental_ref: <true|false>  # default value to be discussed
+```
+
+A registry can import one or more semantic conventions from other published registries. These dependencies
+are declared in the `dependencies` section. The `alias` field is optional but must be specified
+if multiple imported registries share the same `name`. 
+
+Based on the `dependencies` section, Weaver can automatically update dependencies to the latest version when a new
+version becomes available (see the [Registry Update](#registry-update) section). It also allows registry authors to use
+registry names as preferred, without concerns about conflicts.
+
+Sections such as [Registry Resolution](#registry-resolution), [Registry Packaging](#registry-packaging), and 
+[Registry Publication](#registry-publication) provide more details on how the `weaver_registry.yaml` file
+is used in these processes.
+
+Open Questions:
+- Should we follow SemVer 2 for registry versions? It seems advisable, as Weaver can detect breaking changes. However,
+  the resolution process doesn't rely on SemVer 2 in an important way.
+- How do we retrieve a published registry URL from a schema URL? Perhaps a new field in the schema URL file could
+  point to the published registry URL, possibly named `registry_url`.
+
+### Semantic Convention File Structure
+
+The current [JSON Schema](https://github.com/open-telemetry/weaver/blob/main/schemas/semconv.schema.json) that
+describes the files of the semantic convention registry remains mostly unchanged. However, a few minor updates are
+required:
+
+- The ability to reference a group must be supported, allowing a registry to extend or override fields of an
+  imported group.
+- The structure of a reference must be enhanced to disambiguate attribute or signal references from other registries
+  when necessary (more details in the [Registry Resolution](#registry-resolution) section).
+
+Currently, a reference follows dot notation (e.g., `ref: client.address`). When there is no ambiguity or conflict
+detected by Weaver, this notation is sufficient. However, when a conflict occurs, the user must disambiguate the
+reference by prefixing it with the registry name (e.g., `ref: otel:client.address`). The prefix corresponds
+to the registry name as defined in the `weaver_registry.yaml` file under the `dependencies` section. A colon is used
+as a separator between the name and the attribute or group name. References to groups or attributes defined in the
+local registry are never prefixed.
+
+In a group, the `extends` section can now reference an imported group using this new reference syntax. Similarly, the
+`attributes` section of a group can reference an imported attribute.
+
+In the current semantic conventions specification, referencing a group is currently unsupported. Uniqueness within
+groups is scoped by the type of group. It is entirely possible to have an event and a metric identified by the same ID.
```

Comment:
Maybe we should fix this first -  This is something we could do BEFORE allowing multiple registries and I think would make our lives better overall.

cc @lmolkova for thoughts on that.

Reply:
@jsuereth I think it's a feature, not a bug - https://github.com/open-telemetry/weaver/pull/348#discussion_r1742858393

E.g. `http.request.body.size` can in theory be an attribute and a metric name. Or `messaging.message.time_in_queue`. 

I don't believe we have real examples of this but there were some discussions in [the past](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32486#discussion_r1575584907) where it would make sense to use the same attribute name on an event/span as some existing metric name.

It could be a good feature for spans/events->metrics pipeline (take specific attribute and convert it to metric) and I'd like to check if we can keep this door open.